consensus/types/src/chain_spec.rs

```diff
@@ -437,6 +438,7 @@ impl ChainSpec {
             network_id: 1, // mainnet network id
             attestation_propagation_slot_range: 32,
             attestation_subnet_count: 64,
+            sync_committee_subnet_count: 4,
```

Comment:
I think this should be deleted from `ChainSpec` because it's on the `EthSpec`.

We _also_ have a constant `SYNC_COMMITTEE_SUBNET_COUNT` in `consts.rs` which should probably be removed in favour of the `EthSpec` value (although that refactor will be a bit more involved).

Reply:
Yeah I added this because we had `attestation_subnet_count` in the `ChainSpec` and `SubnetBitfieldLength` in the `EthSpec` which is similar to this. I think we could have had it just in the `EthSpec` but wasn't so sure if there was a reason having it in both.

Reply:
Oh right, interesting... I think lets clean up the sync one for now and consider coming back for the attestation subnet count later. Perhaps we could tolerate 2 representations (const and EthSpec), so long as we have a test somewhere that they're consistent?